lib/livebook_web/live/session_live.ex

```diff
@@ -636,19 +652,22 @@ defmodule LivebookWeb.SessionLive do
   end
 
   def handle_event("delete_cell", %{"cell_id" => cell_id}, socket) do
+    assert_policy!(socket, :edit)
```

Comment:
Note for later, we need to make sure to disable the corresponding shortcuts in readonly mode, so they don't crash the LV. This is one more place to keep in sync, so an alternative we could consider is using guards instead of raising an error, so we just ignore invalid actions. I'm not sure yet.